Libraries/src/Amazon.Lambda.RuntimeSupport/Helpers/FileDescriptorLogStream.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+using System.IO;
+using System.Buffers;
+using Microsoft.Win32.SafeHandles;
+using System.Collections.Concurrent;
+
+namespace Amazon.Lambda.RuntimeSupport.Helpers
+{
+    /// <summary>
+    /// This class wraps the utility of writing to the Lambda telemetry file descriptor for logging into a standard .NET Stream.
+    /// The message
+    /// </summary>
+    public static class FileDescriptorLogFactory
+    {
+        static ConcurrentDictionary<string, StreamWriter> _writers = new ConcurrentDictionary<string, StreamWriter>();
```

Comment:
can have restricted access
```suggestion
        private static readonly ConcurrentDictionary<string, StreamWriter> Writers = new();
```

Reply:
Added the `private` and `readonly`. I didn't use `new()` because it isn't supported in the version of C# this library targets which is 7.3.